src/data/sym/card.lean

```diff
@@ -0,0 +1,85 @@
+/-
+Copyright (c) 2021 Yaël Dillies, Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Bhavik Mehta
+-/
+import algebra.big_operators.basic
+import data.sym.sym2
+
+/-!
+# Stars and bars
+
+In this file, we prove the case `k = 2` of stars and bars.
+
+## Informal statement
+
+If we have `n` objects to put in `k` boxes, we can do so in exactly `(n + k - 1).choose n` ways.
+
+## Formal statement
+
+We can identify the `k` boxes with the elements of a fintype `α` of card `k`. Then placing `n`
+elements in those boxes corresponds to choosing how many of each element of `α` appear in a multiset
+of card `n`. `sym α n` being the subtype of `multiset α` of multisets of card `n`, our type stars
+and bars can be written as
```

Comment:
What do you mean by `our type stars and bars`?

Reply:
"Stars and bars written using types"